src/UI/Component/Listing/DescriptiveList.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Component\Listing;
+
+/**
+ * Interface DescriptiveList
+ * @package ILIAS\UI\Component\Listing
+ */
+interface DescriptiveList extends \ILIAS\UI\Component\Component {
+
+    /**
```

Comment:
Thanks and sorry. I yet have to find out how those spaces got mixed in again and again.